recipes/libgit2/all/FindLIBSSH2.cmake

```diff
@@ -0,0 +1,51 @@
+#
```

Comment:
did you try with `cmake_find_package` generator instead? We can update the module name for libssh2 if it's wrong.

Reply:
I don't know of a *correct* name for libssh2. Because there is no central libssh2 cmake recipe, people make up their own name.

Reply:
I removed the need of a custom `FinsLIBSSH2.cmake` by settings the required variables after the `find_package`.